berza/src/main/java/rs/edu/raf/banka/berza/bootstrap/BootstrapData.java

```diff
@@ -68,5 +73,33 @@ public void run(String... args) throws Exception {
             valute.add(v);
         }
         valutaRepository.saveAll(valute);
+
+        //Dodavanje informacija o berzi
+        List<Berza> berze = new ArrayList<>();
+        fileName = "resources/berze.csv";
+
+        List<BerzaCSV> berzeCSV = new CsvToBeanBuilder(new FileReader(fileName))
+                .withType(CurrencyCSV.class)
```

Comment:
Ovo treba da bude BerzaCSV.

```suggestion
                .withType(BerzaCSV.class)
```